Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -601,7 +601,7 @@ def genS(fragment,howMuch):
 steps['AMPT_PPb_5020GeV_MinimumBias']=merge([{'-n':10},step1PPbDefaults,genS('AMPT_PPb_5020GeV_MinimumBias_cfi',Kby(9,100))])
 
 ## pPb Run2
-step1PPbDefaultsUp15={'--beamspot':'Run2PPbBoost','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
+step1PPbDefaultsUp15={'--beamspot':'RealisticPPbBoost8TeV2016Collision','--conditions':'auto:run2_mc_pa','--eventcontent':'RAWSIM', '--era':'Run2_2016_pA'}
```

Comment:
@davidlange6 That's too complicated IMO, as the new pPb vertex smearing is now what's linked from the boost tool.  I would have to undo that, which is the entire point of this PR.  If we must really keep a consistent relval, then I'll just update the autoconditions with the GT when it's ready.